docs/book/releases/v2.0.0.md

```diff
@@ -0,0 +1,94 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.0.0 release
+
+## New Feature
+
+- Offline Persistent Volume expansion for Block Volume.
+- ReadWriteMany volumes using vSAN file services.
+- Support for enabling leader election to ensure High Availability for vSphere CSI driver.
+
+Note: This release is not backward compatible to vSphere 67u3 release.
```

Comment:
Why? This statement is very scary, if there is a minor issue we should call it out and not say that it is not backwards compatible. But, I would to know more on why we are saying this.

Reply:
removed this line and added notes

>    - Workaround: If you are using vSphere 6.7 Update3 with `v2.0.0` release of the driver, it is recommended to use `v2.0.1` release of the driver.